File: module22/module01/module43/src/main/java/com/android/example/module22_module01_module43/ClassAAJ.java
package com.android.example.module22_module01_module43;

public class ClassAAJ {
	private 	com.android.example.module01_module39.ClassAAC instance_var_1_0 = new com.android.example.module01_module39.ClassAAC();
	private 	com.android.example.module01_module39.ClassAAJ instance_var_1_1 = new com.android.example.module01_module39.ClassAAJ();
	private 	com.android.example.module01_module39.ClassAAB instance_var_1_2 = new com.android.example.module01_module39.ClassAAB();
	private 	com.android.example.module01_module39.ClassAAI instance_var_1_3 = new com.android.example.module01_module39.ClassAAI();

	public void method0(
		com.android.example.module01_module39.ClassAAF param0,
		com.android.example.module01_module39.ClassAAE param1,
		com.android.example.module01_module39.ClassAAB param2,
		com.android.example.module01_module39.ClassAAB param3) throws Throwable {
	}

	public void method1(
		com.android.example.module01_module39.ClassAAJ param0) throws Throwable {
		io.reactivex.internal.disposables.ListCompositeDisposable local_var_2_1 = new io.reactivex.internal.disposables.ListCompositeDisposable();
		local_var_2_1.isDisposed();

		if (new java.lang.Object().equals(new java.lang.Object())) {
			for (int iAb = 0; iAb < 0; iAb++) {
				io.reactivex.subscribers.TestSubscriber<Object> local_var_4_0 = new io.reactivex.subscribers.TestSubscriber();
				local_var_4_0.isCancelled();

				local_var_4_0.onError(new java.lang.Throwable());

				io.reactivex.internal.disposables.ListCompositeDisposable local_var_4_1 = new io.reactivex.internal.disposables.ListCompositeDisposable();
				local_var_4_1.isDisposed();

			}
		} else {
			io.reactivex.internal.util.LinkedArrayList local_var_3_0 = new io.reactivex.internal.util.LinkedArrayList(77);
			local_var_3_0.head();

		}
		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_2 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_2.assertNotSubscribed();

		param0.method1(new io.reactivex.subscribers.TestSubscriber(), new io.reactivex.subscribers.TestSubscriber(), new io.reactivex.subscribers.TestSubscriber(), new io.reactivex.internal.util.OpenHashSet());

	}

	public void method2(
		com.android.example.module01_module39.ClassAAH param0) throws Throwable {
		io.reactivex.subscribers.TestSubscriber<Object> local_var_2_1 = new io.reactivex.subscribers.TestSubscriber();
		local_var_2_1.isDisposed();

	}

	public void method3(
		com.android.example.module01_module39.ClassAAD param0,
		com.android.example.module01_module39.ClassAAE param1,
		com.android.example.module01_module39.ClassAAI param2) throws Throwable {
		param0.method0(new io.reactivex.subscribers.TestSubscriber(), new io.reactivex.subscribers.TestSubscriber(), new io.reactivex.subscribers.TestSubscriber());

	}
}

File: module04/module03/module3/src/main/java/com/android/example/module04_module03_module3/ClassAAJ.kt
package com.android.example.module04_module03_module3

@Suppress("UNUSED_PARAMETER")
class ClassAAJ {
	private 	val instance_var_1_0: com.android.example.module04_module12_module6.ClassAAH = com.android.example.module04_module12_module6.ClassAAH()
	private 	val instance_var_1_1: com.android.example.module04_module12_module6.ClassAAA = com.android.example.module04_module12_module6.ClassAAA()

	fun method0(
		param0: com.android.example.module04_module12_module6.ClassAAE,
		param1: com.android.example.module04_module12_module6.ClassAAA,
		param2: com.android.example.module04_module12_module6.ClassAAH,
		param3: com.android.example.module04_module12_module6.ClassAAH
	) {
		val local_var_2_4: io.reactivex.internal.disposables.ListCompositeDisposable = io.reactivex.internal.disposables.ListCompositeDisposable()
		local_var_2_4.isDisposed()

	}

	fun method1(
		param0: com.android.example.module04_module12_module6.ClassAAA,
		param1: com.android.example.module04_module12_module6.ClassAAD
	) {
		if (kotlin.Any().equals(kotlin.Any())) {
			for (iAb in 0..0) {
				val local_var_4_0: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
				local_var_4_0.onNext(kotlin.Any())

				val local_var_4_1: io.reactivex.internal.util.AppendOnlyLinkedArrayList<kotlin.Any> = io.reactivex.internal.util.AppendOnlyLinkedArrayList(25)
				local_var_4_1.add(kotlin.Any())

			}
		} else {
			val local_var_3_0: io.reactivex.internal.schedulers.SingleScheduler = io.reactivex.internal.schedulers.SingleScheduler()
			local_var_3_0.shutdown()

		}
		val local_var_2_2: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_2.assertNotSubscribed()

	}

	fun method2(
		param0: com.android.example.module04_module12_module6.ClassAAF,
		param1: com.android.example.module04_module12_module6.ClassAAI,
		param2: com.android.example.module04_module12_module6.ClassAAF
	) {
		val local_var_2_3: io.reactivex.disposables.SerialDisposable = io.reactivex.disposables.SerialDisposable()
		local_var_2_3.get()

		param0.method0("SomeString", "SomeString", "SomeString")

	}

	fun method3(
		param0: com.android.example.module04_module12_module6.ClassAAG
	) {
		param0.method2("SomeString", "SomeString", "SomeString")

	}

	fun method4(
		param0: com.android.example.module04_module12_module6.ClassAAC
	) {
		param0.method2("SomeString", "SomeString", "SomeString")

		val local_var_2_1: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_1.isCancelled()

		val local_var_2_2: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()
		local_var_2_2.assertSubscribed()

	}
}
